### PR TITLE
Update crd.md

### PR DESCRIPTION
**Pull request by moonsub-kim**
I have updated crd.md with more readable format and added missing options from flinkcluster_types.go

### CODE REVIEW

File: docs/crd.md [hunk 1]
```diff
@@ -7,399 +7,183 @@ whether the job spec is specified. Similarly to other kinds of Kubernetes resour
...
-      - **triggerReason**: Savepoint triggered reason.
-      - **state**: Savepoint state.
-      - **message**: Savepoint message.
-    - **currentRevision**: CurrentRevision indicates the version of FlinkCluster.
-    - **nextRevision**: NextRevision indicates the version of FlinkCluster updating.
-    - **collisionCount**: collisionCount is the count of hash collisions for the FlinkCluster.
-      The controller uses this field as a collision avoidance mechanism
-      when it needs to create the name for the newest ControllerRevision.
-    - **lastUpdateTime**: Last update timestamp of this status.
+| Key | Type | Optional | Description
```
Comment (regadas) [inline docs/crd.md:13]:
```suggestion
| Key | Type | Required | Description
```

and then we could only mark the required fields. wdyt?

**Review by regadas — COMMENTED** [review]
I think this really makes it more readable; thanks for doing this @moonsub-kim

File: docs/crd.md (hunk 1 above)
Comment (moonsub-kim) [reply docs/crd.md:13]: good! XOXOXOXXX make us confuse

Comment (moonsub-kim) [reply docs/crd.md]: https://github.com/moonsub-kim/flink-on-k8s-operator/blob/crd/docs/crd.md

**Review by regadas — APPROVED** [review]

**Comment by regadas** [comment]
Thanks @moonsub-kim

**Pull request merged**